go-controller/pkg/ovn/policy.go

```diff
@@ -222,8 +243,100 @@ func (oc *Controller) deleteACLAllow(namespace, policy, logicalSwitch,
 	}
 }
 
-func (oc *Controller) addACLDeny(namespace, logicalSwitch,
-	logicalPort, policyType string) {
+func (oc *Controller) addIPBlockACLDeny(namespace, policy, logicalSwitch,
+	logicalPort, policyType, except, priority string) {
+	var match, l3Match, direction, portMatch string
+	if policyType == ingressPolicyType {
+		portMatch = fmt.Sprintf("outport == \\\"%s\\\"", logicalPort)
+		l3Match = fmt.Sprintf("ip4.src == %s", except)
+		match = fmt.Sprintf("match=\"%s && %s\"", portMatch, l3Match)
+		direction = toLport
+	} else {
+		portMatch = fmt.Sprintf("inport == \\\"%s\\\"", logicalPort)
+		l3Match = fmt.Sprintf("ip4.dst == %s", except)
+		match = fmt.Sprintf("match=\"%s && %s\"", portMatch, l3Match)
+		direction = fromLport
+	}
+
+	uuid, err := exec.Command(OvnNbctl, "--data=bare", "--no-heading",
+		"--columns=_uuid", "find", "ACL", match, "action=drop",
+		fmt.Sprintf("external-ids:default-deny-direction=%s", direction),
```

Comment:
We should change this name from "default-deny-direction" to ipblock-deny-direction. Same in couple of other places.

Reply:
Good point, updated to ipblock-deny-direction.